loki/package.json

```
@@ -0,0 +1,96 @@
+{
+  "name": "@perses-dev/loki-plugin",
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "rsbuild dev",
+    "build": "npm run build-mf && concurrently \"npm:build:*\"",
+    "build-mf": "rsbuild build",
+    "build:cjs": "swc ./src -d dist/lib/cjs --strip-leading-paths --config-file .cjs.swcrc",
+    "build:esm": "swc ./src -d dist/lib --strip-leading-paths --config-file .swcrc",
+    "build:types": "tsc --project tsconfig.build.json",
+    "test": "cross-env LC_ALL=C TZ=UTC jest",
+    "type-check": "tsc --noEmit"
+  },
+  "main": "lib/cjs/index.js",
+  "module": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "devDependencies": {
+    "@module-federation/rsbuild-plugin": "^0.11.2",
+    "@rsbuild/core": "^1.2.19",
+    "@rsbuild/plugin-react": "^1.1.1",
+    "@types/react-dom": "^18.3.0",
+    "@types/jest": "^29.5.14",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-router-dom": "^6.26.0",
+    "concurrently": "^9.1.2",
+    "jest": "^29.7.0",
+    "@swc/cli": "^0.6.0",
+    "@swc/core": "^1.7.10",
+    "@swc/jest": "^0.2.37",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^16.1.0",
+    "@testing-library/user-event": "^13.5.0"
+  },
+  "peerDependencies": {
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
+    "@hookform/resolvers": "^3.2.0",
+    "@perses-dev/components": "^0.51.0-beta.1",
+    "@perses-dev/core": "^0.51.0-beta.1",
+    "@perses-dev/dashboards": "^0.51.0-beta.1",
+    "@perses-dev/explore": "^0.51.0-beta.1",
+    "@perses-dev/plugin-system": "^0.51.0-beta.1",
+    "@tanstack/react-query": "^4.39.1",
+    "date-fns": "^4.1.0",
+    "date-fns-tz": "^3.2.0",
+    "echarts": "5.5.0",
+    "immer": "^10.1.1",
+    "lodash": "^4.17.21",
+    "react": "^17.0.2 || ^18.0.0",
+    "react-dom": "^17.0.2 || ^18.0.0",
+    "react-hook-form": "^7.52.2",
+    "use-resize-observer": "^9.0.0"
+  },
+  "files": [
+    "lib/**/*",
+    "__mf/**/*",
+    "mf-manifest.json",
+    "mf-stats.json"
+  ],
+  "perses": {
+    "moduleName": "Loki",
+    "moduleOrg": "loki-org",
+    "schemasPath": "schemas",
+    "plugins": [
+      {
+        "kind": "Datasource",
+        "spec": {
+          "display": {
+            "name": "Loki Datasource"
+          },
+          "name": "LokiDatasource"
+        }
+      }
+      ,{
+        "kind": "TimeSeriesQuery",
+        "spec": {
+          "display": {
+            "name": "Loki Query"
+          },
+          "name": "LokiQuery"
+        }
+      }
+      ,{
+        "kind": "Panel",
+        "spec": {
+          "display": {
+            "name": "Logs"
+          },
+          "name": "Logs"
```

Maybe LogsTable will be better

```suggestion
          "display": {
            "name": "LogsTable"
          },
          "name": "LogsTable"
```